dart/lib/src/feature_flags_integration.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:async';
+
+import 'hub.dart';
+import 'integration.dart';
+import 'sentry_options.dart';
+import 'protocol/sentry_feature_flags.dart';
+import 'protocol/sentry_feature_flag.dart';
+
+/// Integration which handles adding feature flags to the scope.
+class FeatureFlagsIntegration extends Integration<SentryOptions> {
+  Hub? _hub;
+
+  @override
+  void call(Hub hub, SentryOptions options) {
+    _hub = hub;
+    options.sdk.addIntegration('featureFlagsIntegration');
```

Comment:
I think we should align future integrations with the naming, I'd opt to go for a capitalized starting letter `FeatureFlagsIntegration`